letterparser/parse.py

```diff
@@ -1,4 +1,38 @@
-import pypandoc
+import os
+import docker
+from letterparser.utils import unicode_encode, unicode_decode
 
-def dummy():
-    return True
+
+DOCKER_IMAGE = "knsit/pandoc:v2.7"
+
+
+def get_docker_client():
+    return docker.from_env()
+
+
+def get_docker_volume_and_file(file_name):
+    parts = file_name.split(os.sep)
+    file_name_path = os.sep.join(parts[0:-1])
+    file_name_file = parts[-1]
+    return file_name_path, file_name_file
+
+
+def get_docker_volume(source_path, bind_path='/data', mode='ro'):
+    return {source_path: {'bind': bind_path, 'mode': mode}}
+
+
+def call_pandoc(file_name, output_format="jats"):
+    client = get_docker_client()
+    file_name_path, file_name_file = get_docker_volume_and_file(file_name)
+    volumes = get_docker_volume(file_name_path)
+    command = 'pandoc "/data/%s" -t %s' % (file_name_file, output_format)
+    output = client.containers.run(DOCKER_IMAGE, command, volumes=volumes)
+    return output
+
+
+def raw_jats(file_name, root_tag="root"):
+    "convert file content to JATS"
```

Comment:
```suggestion
   """convert file content to JATS"""
```

Reply:
I'm going to apply these in a single git commit rather than trying the Github "Commit suggestion", which might create a commit for each - thanks!

Reply:
Correct, it would. Good idea.